src/Microsoft.PowerShell.Commands.Utility/commands/utility/ShowMarkdownCommand.cs

```diff
@@ -28,46 +30,119 @@ public class ShowMarkdownCommand : PSCmdlet
         /// Gets or sets InputObject of type Microsoft.PowerShell.MarkdownRender.MarkdownInfo to display.
         /// </summary>
         [ValidateNotNullOrEmpty]
-        [Parameter(Mandatory = true, ValueFromPipeline = true)]
+        [Parameter(Mandatory = true, ValueFromPipeline = true, ParameterSetName = "InputObject")]
         public PSObject InputObject { get; set; }
 
+        /// <summary>
+        /// Gets or sets path to markdown file(s) to display.
+        /// </summary>
+        [ValidateNotNullOrEmpty]
+        [Parameter(Position = 0, Mandatory = true,
+                   ValueFromPipelineByPropertyName = true, ParameterSetName = "Path")]
+        public string[] Path { get; set; }
+
+        /// <summary>
+        /// Gets or sets the literal path parameter to markdown files(s) to display.
+        /// </summary>
+        [Parameter(ParameterSetName = "LiteralPath",
+                   Mandatory = true, ValueFromPipeline = false, ValueFromPipelineByPropertyName = true)]
+        [Alias("PSPath", "LP")]
+        public string[] LiteralPath
+        {
+            get { return Path; }
+            set { Path = value; }
+        }
+
         /// <summary>
         /// Gets or sets the switch to view Html in default browser.
         /// </summary>
         [Parameter]
         public SwitchParameter UseBrowser { get; set; }
 
-        private SteppablePipeline stepPipe;
+        private System.Management.Automation.PowerShell _powerShell;
 
         /// <summary>
         /// Override BeginProcessing.
         /// </summary>
         protected override void BeginProcessing()
         {
-            if (!UseBrowser.IsPresent)
-            {
-                // Since UseBrowser is not bound, we use proxy to Out-Default
-                stepPipe = ScriptBlock.Create(@"Microsoft.PowerShell.Core\Out-Default @PSBoundParameters").GetSteppablePipeline(this.MyInvocation.CommandOrigin);
-                stepPipe.Begin(this);
-            }
+            _powerShell = System.Management.Automation.PowerShell.Create();
```

Comment:
This constructor will create a new Runspace. Please use `PowerShell.Create(RunspaceMode.CurrentRunspace)` instead, which will use a nested pipeline in the same Runsapce.